### PR TITLE
Bump alpine to 3.19

### PR DESCRIPTION
**Pull request by wallyqs**

### CODE REVIEW

File: 2.10.x/alpine3.19/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.18
+FROM alpine:3.19
 
 ENV NATS_SERVER 2.10.7
```
Comment (derekcollison) [inline 2.10.x/alpine3.19/Dockerfile:3]: Should this be 2.10.8?

File: tests/build-images.sh [hunk 2]
```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 set -ex
 
-ver=(NATS_SERVER 2.10.7)
+ver=(NATS_SERVER 2.9.24)
```
Comment (derekcollison) [inline tests/build-images.sh:4]: Why this vs 2.10.8?

File: tests/run-images.sh [hunk 3]
```diff
@@ -4,7 +4,7 @@ set -ex
 ver=(NATS_SERVER 2.10.7)
```
Comment (derekcollison) [inline tests/run-images.sh:4]: 2.10.8?

File: tests/build-images.sh (hunk 2 above)
Comment (bruth) [reply tests/build-images.sh:4]: This PR is to only bump Alpine. The 2.10.8 bump happens in a subsequent PR.

File: tests/run-images.sh [hunk 4]
```diff
@@ -18,7 +18,7 @@ done
...
```
Comment (derekcollison) [inline tests/run-images.sh:18]: Same here?

File: 2.10.x/alpine3.19/Dockerfile (hunk 1 above)
Comment (wallyqs) [reply 2.10.x/alpine3.19/Dockerfile:3]: yes I will update the version in this PR again once the binary is out

**Review by bruth — APPROVED** [review]

**Pull request merged**